TransferMethod/Sources/Coordinators/TransferMethodCoordinators.swift

```diff
@@ -69,7 +70,7 @@ public class SelectTransferMethodTypeCoordinator: NSObject, HyperwalletCoordinat
         self.parentController = parentController
     }
 }
-
+/// AddTransferMethodCoordinator
```

Comment:
Update to - /// Coordinator class for AddTransferMethod

Reply:
Updated